client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -853,6 +853,7 @@ public void shutdown() {
                     this.scheduledExecutorService.shutdown();
                     this.mQClientAPIImpl.shutdown();
                     this.rebalanceService.shutdown();
+                    this.defaultMQProducer.shutdown();
```

Comment:
`this.defaultMQProducer.shutdown();` and `this.defaultMQProducer.getDefaultMQProducerImpl().shutdown(false);`  conflict?

Reply:
oh, it is conflict ,I delete the last line